Python/bytecodes.c

```diff
@@ -4072,6 +4072,11 @@ dummy_func(
             TIER_TWO_ONLY
             value = ptr;
         }
+        pure op (_POP_TOP_LOAD_CONST_INLINE_BORROW, (ptr/4, pop -- value)) {
+            TIER_TWO_ONLY;
+            DECREF_INPUTS();
+            value = ptr;
+        }
```

Comment:
Maybe there could be an even more specialized opcode that doesn't DECREF its input? That would be handy for the case where you introduced this -- the input is known to be None, so we don't really need to bother with the DECREF of the immortal value.

Reply:
The expectation is that the `_POP_TOP` part will be removed in a later pass. 
`LOAD_CONST ; _POP_TOP_LOAD_CONST_INLINE_BORROW` -> `_LOAD_CONST_INLINE_BORROW`